### PR TITLE
chore(deps): Update

### PR DESCRIPTION
**Pull request by YXL76**
Update all deps and move `raw-window-handle ` to `dev-dependencies`

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -34,7 +34,7 @@ dbus-crossroads = "0.5.0"
 winit = "0.26.1"
 
 [target.'cfg(target_os = "windows")'.dev-dependencies]
-raw-window-handle = "0.5.0"
+raw-window-handle = "0.4.2"
```
Comment (Sinono3) [inline Cargo.toml:37]: Why not use `0.5.0`?

File: src/platform/windows/mod.rs [hunk 2]
```diff
@@ -63,7 +64,7 @@ impl MediaControls {
     /// Attach the media control events to a handler.
     pub fn attach<F>(&mut self, event_handler: F) -> Result<(), Error>
     where
-        F: Fn(MediaControlEvent) + Send + 'static,
+        F: Fn(MediaControlEvent) + Send + Sync + 'static,
```
Comment (Sinono3) [inline src/platform/windows/mod.rs:67]: Why was this Sync clause added? The interface must remain `F: Fn(MediaControlEvent) + Send + 'static` on all systems.

File: Cargo.toml (hunk 1 above)
Comment (YXL76) [reply Cargo.toml:37]: `0.5.0` has some breaking changes, it need to be consistent with `winit`.

https://github.com/rust-windowing/winit/blob/430a49ebc2e1dbe7a8e06518e52bd5de7f33f2e6/Cargo.toml#L48

File: src/platform/windows/mod.rs (hunk 2 above)
Comment (YXL76) [reply src/platform/windows/mod.rs:67]: Refer to https://github.com/microsoft/windows-rs/commit/afe32525c22209aa8f632a0f4ad607863b51796a#diff-8e5c79df437470375e1139a935230234932ca20cb0fba4dcc8f664b53e7e0b34R4416, The event_handler need to be `Send`, so `F` need to be `Send` and `Sync` (https://github.com/rust-lang/rust/blob/f8588549c3c3d45c32b404210cada01e2a45def3/library/alloc/src/sync.rs#L241).

Comment (YXL76) [reply src/platform/windows/mod.rs]: Fixed.

**Comment by Sinono3** [comment]
Thank you! Merging :)

**Pull request merged**